server/lib/system/system.createContainer.js

```diff
@@ -0,0 +1,21 @@
+const { PlatformNotCompatible } = require('../../utils/coreErrors');
+
+/**
+ * @description Pull an new container image.
+ * @param {string} options - Otpions.
+ * @returns {Promise} THe created container.
+ * @example
+ * await createContainer(options);
+ */
+async function createContainer(options) {
+  if (!this.dockerode) {
+    throw new PlatformNotCompatible('SYSTEM_NOT_RUNNING_DOCKER');
+  }
+  const createdContainer = await this.dockerode.createContainer(options);
+  const containers = await this.getContainers({ all: true, filters: { id: [createdContainer.id] } });
```

Comment:
I think here docker container need to be started if successfully created

Reply:
The container is just created here, not started.
It will be started at the MQTT save configuration step.